### ISSUE TITLE
I have a bootable OC 0.7.3 EFI

### ISSUE THREAD

**Issue by zhujinle** [Necromancer2Point0/MSI-GS66-OC-New-Dawn#2]
https://github.com/zhujinle/MSI-GS66-Stealth-OC-Hackintosh

**Comment by Necromancer2Point0** [comment]
Hi, thanks for the link!

I've tinkered around with my work in progress EFI in accordance with your files via OC Config Compare, modifying it to use my kexts etc. After verifying with OC Validate, everything looks to be in order.

However, I have **not tried to boot** with it yet (busy at the moment) but have uploaded my EFI (with redacted smbios info) to my "WiP" directory in case you want to conduct your own experiments. I will try the EFI out myself pretty soon.

**Comment by Necromancer2Point0** [comment]
Tried booting with my new build to no avail. Getting 0xE errors amongst other problems.

Minor update to the files with newer kexts but as I said, not bootable using my WiP build.

**Comment by zhujinle** [comment]
Oh, sorry. I'm a student from China and working for my paper these days. I'm so busy that I forget to tell you that I've disabled my PM981a SSD with SSDT in my EFI, which file was called 'SSDT-PM981a-Disable.aml'. If your machine does not have the same SSD in the first SSD slot or if the problem was caused by this SSDT, please delete it. I will test your EFI in my GS66-664 these days. By the way, My screen will flicker in 60Hz mode, how can I fix it? I've tried SwitchResX but I can't active the settings.  :(

**Comment by zhujinle** [comment]
I’ve tested your EFI, and absolutely get several panic errors. So I removed some of your Kexts and changed some settings. For example, you added VodooInput.kext for 3 times, which Wii cause the kernel panic. So I delete VodooInpot.kext, VoodooI2CAtmelMXT.kext, VoodooI2CELAN.kext, VoodooI2CFTE.kext, VoodooHDA.kext(some of them have VodooInput as plugins and I don't want to find the keys in plist so I just delete them and others is because I think it's useless). Then I tried use Kernel-Quirks: AppleCpuPmCfgLock, AppleXcpmCfgLock as my EFI does. Then it become bootable but have some small bugs. I can't tell you whether Ethernet is useable, because I can't connect my ISP by PPPOE. But I can tell you that I've tested that in my EFI and it works well.

**Comment by Necromancer2Point0** [comment]
Hi,

I have been using my latest stable build (V 0.5) on an earlier version of OpenCore (0.6.0) with no issues whatsoever. Everything is working perfectly though I have not tested bluetooth, ethernet, or thunderbolt. 

I actually tried cutting down on the voodoo kexts based on your EFI but found that the trackpad became unusable with the same combination. I have no issues on my stable build with all these kexts enabled and keyboard + trackpad functionality is excellent.

I use VoodooHDA for sound as AppleALC either gives good speaker audio or good headphone jack audio but not BOTH, no matter what alcid I used. I highly recommend trying out VoodooHDA if sound is important to you.

My "work in progress" experimental build was meant for 0.7.3 but whilst OC Validate and other tools say that it should be okay on paper, it was still unbootable for me.

It's interesting that you were able to boot after deleting the voodoo kexts. Maybe something changed with the newest OC release.

Even so, enabling those kernel quirks did not fix the issue for me when I was experimenting and is not really recommended in the long term as it interferes with native CPU power management, decreasing overall battery life and efficiency.

For me, my current stable build suits all my needs and I don't really mind so much about being on an older version of OC.

As for the screen flickering, I have never encountered that issue so I'm not sure where that stems from. Originally, my smbios settings were incorrect (think it was set as an iMac) and I remember being able to change my refresh rate to 300hz directly in system preferences.

After correcting the smbios info however, this refresh rate setting disappeared. I completely forgot about it until you just mentioned your 60hz issue but may look into verifying what my current refresh rate is and looking into possible adjustments to the refresh rate of this build in the future.

Anyways, my point is that if you need something stable now and don't mind running an older version of OC, I highly recommend looking at my stable build.

If you really care about being on the newest OC release, then by all means continue to experiment. It would be really nice to have a stable build on the newest version of OC without all the panics and bugs so please let me know if you have any success :) 

I'm quite busy too nowadays and honestly am satisfied with the stability of my current build so probably will not be as active in tinkering with newer releases of OC at the moment.

All the best!

**Comment by zhujinle** [comment]
Hi,
In fact, you only need to delete the repeatedly loaded VodooInput.kext. You don’t need to delete the kext like I did. I just want to make sure that the system can be started. The only need to do is to find the repeatedly loaded VodooInput.kext and delete it from plist.
I've tried your EFI but for some reason I can't use PPPoe to connect to my Internet. So I just come back to my EFI. The reason why I use the newest OC is because I found a copy of the old version of EFI, but there are many problems. Anyway, I have to redo it, so I just made the most detailed version.
For VoodooHDA, I will try it sometime, I found the sound quality of my headphones is poor, maybe it will help.
As for the screen flickering, it's still happen in your EFI, I think maybe is my machine's problem, hope someone can give me the solution.
Thanks for your help!
